cmd/compute-domain-kubelet-plugin/nvlib.go

```diff
@@ -75,12 +77,29 @@ func newDeviceLib(driverRoot root) (*deviceLib, error) {
 	nvmllib := nvml.New(
 		nvml.WithLibraryPath(driverLibraryPath),
 	)
+
+	mic, err := getImexChannelCount()
+	if err != nil {
+		return nil, fmt.Errorf("error getting max IMEX channel count: %w", err)
+	}
```

Comment:
Can we hang this back off of the deviceLib object. I realize this backgracks from an earlier comment, but on further thought, I think this is better. The methods hanging off of deviceLib are meant to "eventually" make their way into the actual go-nvlib repo.